torch/testing/_internal/common_utils.py

```diff
@@ -1353,7 +1353,7 @@ def download_file(url, binary=True):
     except error.URLError:
         msg = "could not download test file '{}'".format(url)
         warnings.warn(msg, RuntimeWarning)
-        raise unittest.SkipTest(msg)
+        raise unittest.SkipTest(msg) from None
```

Comment:
IMO exception chaining would be useful here, wouldn't it?

Reply:
As `torch/testing/` module is for testing, your suggestion sounds reasonable.